### ISSUE TITLE
is this repo maintained  ?

### ISSUE THREAD

**Issue by equivalent** [cortexinsight/shrine-azure#1]
just asking as Readme is generated, it has full of TODO, gemspec has full of TODO, `gem install shrine-azure` doesn't work, it have 0 tests yet somehow it manage to get on https://shrinerb.com/ official page 

are you planing to maintain it or should developers consider write own solution / fork this and continue from there ?

**Comment by mort666** [comment]
Yes and no, it was developed to support an app migration to Azure which didn't happen. I've no idea how it happened to get on the shrinerb homepage.

At the moment we're focusing on a couple of other things but we will clean things up in here as it is very much a WIP.. Happy to accept pull requests and issues.
